tests/Frameworks/Laravel/Version_8_x/routes/web.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\CommonSpecsController;
+use App\Http\Controllers\EloquentTestController;
+
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('simple', [CommonSpecsController::class, 'simple'])->name('simple_route');
+Route::get('simple_view', [CommonSpecsController::class, 'simple_view']);
+Route::get('error', [CommonSpecsController::class, 'error']);
+Route::get('eloquent/get', [EloquentTestController::class, 'get']);
+Route::get('eloquent/insert', [EloquentTestController::class, 'insert']);
+Route::get('eloquent/update', [EloquentTestController::class, 'update']);
+Route::get('eloquent/delete', [EloquentTestController::class, 'delete']);
+Route::get('eloquent/destroy', [EloquentTestController::class, 'destroy']);
+Route::get('eloquent/refresh', [EloquentTestController::class, 'refresh']);
```

Comment:
Does this support closures?
What about `[new CommonSpecsController(), 'simple']`?

Reply:
Yes, it [supports closures](https://laravel.com/docs/8.x/routing#basic-routing). But when I tried using an instance of the controller I got this error:

```
production.ERROR: Object of class App\Http\Controllers\CommonSpecsController could not be converted to string
```

Reply:
So, it doesn't support `callable` generally; just specific forms of callables. I suppose we should add tests for the other forms. I'm not aware of a JIRA for this; are you?

Reply:
There wasn't one so I created one. :)